api/api.go

```diff
@@ -39,12 +40,10 @@ func BaseURL() (u url.URL) {
 // request does the heavy lifting of making requests to the Section API.
 //
 // You can pass 0 or more headers, and keys in the later headers will override earlier passed headers.
-func request(method string, u url.URL, body io.Reader, headers ...http.Header) (resp *http.Response, err error) {
-	client := &http.Client{
-		Timeout: Timeout,
-	}
+func request(ctx context.Context, method string, u url.URL, body io.Reader, headers ...http.Header) (resp *http.Response, err error) {
+	client := http.Client{}
```

Comment:
(nitpick) Default value works fine.
```go
var client http.Client
```

Reply:
Also, why don't you reuse the client across different quests?

Reply:
Yep that's a good point about the default value, I don't know why I didn't do it.

Reply:
@pavelnikolov I wasn't re-using the client because I hadn't thought of it! 

I hadn't thought of it because I was setting up a client with some default values (like timouts) on every call to `request()`

Now we're using the default value, and timeouts are handled inside `context`s, 14fbeac96f8b92b24c73ea6121a9d957c6cff5bf moves `client` to a package level variable, so it can be reused.